### PR TITLE
Bug 1949889: Enables text input for volume count in backingstore page

### PR DESCRIPTION
**Pull request by vbnrh**
Signed-off-by: Vineet Badrinath <vbadrina@redhat.com>

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/noobaa-provider-endpoints/pvc-endpoint-type.tsx [hunk 1]
```diff
@@ -68,6 +68,8 @@ export const PVCType: React.FC<PVCTypeProps> = ({ state, dispatch }) => {
             <MinusIcon onClick={substract} />{' '}
           </InputGroupText>
           <TextInput
+            type="number"
+            onChange={(value) => dispatch({ type: 'setVolumes', value: parseInt(value, 10) })}
             value={state.numVolumes}
             aria-label={t('ceph-storage-plugin~Number of Volumes')}
           />
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/components/noobaa-provider-endpoints/pvc-endpoint-type.tsx:75]: Use `NumberInput` PF component.

**Review by bipuladh — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/src/components/noobaa-provider-endpoints/pvc-endpoint-type.tsx (hunk 1 above)
Comment (vbnrh) [reply frontend/packages/ceph-storage-plugin/src/components/noobaa-provider-endpoints/pvc-endpoint-type.tsx]: done

Comment (vbnrh) [reply frontend/packages/ceph-storage-plugin/src/components/noobaa-provider-endpoints/pvc-endpoint-type.tsx]: good call

File: frontend/packages/ceph-storage-plugin/src/components/noobaa-provider-endpoints/pvc-endpoint-type.tsx [hunk 2]
```diff
@@ -63,20 +58,16 @@ export const PVCType: React.FC<PVCTypeProps> = ({ state, dispatch }) => {
...
-        </InputGroup>
+        <NumberInput
+          value={state.numVolumes}
+          onChange={onVolumeChange}
+          onMinus={() => dispatch({ type: 'setVolumes', value: state.numVolumes - 1 })}
+          onPlus={() => dispatch({ type: 'setVolumes', value: state.numVolumes + 1 })}
+          inputName="volume-input"
+          inputAriaLabel="volumes-input"
+          minusBtnAriaLabel="minus"
+          plusBtnAriaLabel="plus"
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/components/noobaa-provider-endpoints/pvc-endpoint-type.tsx:69]:
```suggestion
          plusBtnAriaLabel="plus"
          min={1}
```

**Review by bipuladh — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/src/components/noobaa-provider-endpoints/pvc-endpoint-type.tsx (hunk 2 above)
Comment (vbnrh) [reply frontend/packages/ceph-storage-plugin/src/components/noobaa-provider-endpoints/pvc-endpoint-type.tsx:69]: done

**Comment by bipuladh** [comment]
Tested working as expected.
/lgtm

**Review by bipuladh — APPROVED** [review]
/lgtm

**Pull request merged**